### PR TITLE
[11.0][ADD] web_tree_resize_column: Resize columns in tree views

### PR DESCRIPTION
**Pull request by tarteo**
With this module you can resize the columns of tree views including ones that are nested in form views.

### CODE REVIEW

File: web_tree_resize_column/static/lib/resizableColumns/jQuery.resizableColumns.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+(function($) {
+    $.fn.resizableColumns = function() {
+      var isColResizing = false;
+      var resizingPosX = 0;
+      var _table = $(this);
+      var _thead = $(this).find('thead');
+
+      _thead.find('th').each(function() {
+        $(this).css('position', 'relative');
+        $(this).innerWidth($(this).innerWidth());
```
Comment (Tardo) [inline web_tree_resize_column/static/lib/resizableColumns/jQuery.resizableColumns.js]: Why need this?

Comment (tarteo) [reply web_tree_resize_column/static/lib/resizableColumns/jQuery.resizableColumns.js]: Don't know, this is of the plugin. So ask the author Jones Vinoth Joseph :smile: .

File: web_tree_resize_column/static/src/js/backend.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                styles.push($(this).attr('style'));
+            });
+
+            var res = this._super.apply(this, arguments);
+
+            // Initialize jQuery plugin
+            this.$el.find('table').resizableColumns();
+
+            // Restore width of columns
+            this.$el.find('thead th').each(function (index, th) {
```
Comment (Tardo) [inline web_tree_resize_column/static/src/js/backend.js:27]: Can't reuse previous "find"?

**Comment by Tardo** [comment]
I have this issue when try resize the first time
issue

Firefox 65.0 - Linux

**Review by elicoidal — APPROVED** [review]
LGTM

**Comment by tarteo** [comment]
@Tardo Let me see if I can fix that.

File: web_tree_resize_column/static/src/js/backend.js (hunk 2 above)
Comment (tarteo) [reply web_tree_resize_column/static/src/js/backend.js:27]: No, because after the super the element does not exists anymore.

**Comment by tarteo** [comment]
@Tardo I fixed the issue in Firefox but it was an issue in the plugin. I'm gonna make a PR for that in the plugin's repository.

**Comment by Tardo** [comment]
Yep, now works fine.

**Review by HaGuesto — COMMENTED** [review]
LTGM nice feature!

Just as a comment before I approve:
Is it possible to add some kind of separating lines between the columns? Otherwise, one has to guess where one needs to click.

Before they were shown:
bildschirmfoto_2019-02-08_14-20-22

**Comment by tarteo** [comment]
@HaGuesto I think the lines between the columns is enough, because when the line is hovered the cursor changes to the horizontal resize icon (col-resize)

**Review by HaGuesto — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@Tardo waiting your validation for merging.

**Review by Tardo — APPROVED** [review]

**Pull request merged**